src/applications/disability-benefits/526EZ/config/form.js

```diff
@@ -679,119 +876,90 @@ const formConfig = {
                 type: 'array',
                 items: {
                   type: 'object',
-                  properties: {},
+                  required: ['providerFacility'],
+                  properties: {
+                    providerFacility: {
+                      type: 'array',
+                      minItems: 1,
+                      maxItems: 100,
+                      items: {
+                        type: 'object',
+                        required: [
+                          'providerFacilityName',
+                          'providerFacilityAddress',
+                        ],
+                        properties: {
+                          providerFacilityName: {
+                            type: 'string',
+                            pattern: '^(.{1,100})$',
+                          },
+                          treatmentDateRange: {
+                            type: 'object',
+                            properties: {
+                              from: {
+                                $ref: '#/definitions/date',
+                              },
+                              to: {
+                                $ref: '#/definitions/date',
+                              },
+                            },
+                            required: ['from', 'to'],
+                          },
+                          providerFacilityAddress: Object.assign(
+                            addressSchema(fullSchema4142, true),
+                            {
+                              properties: {
+                                street: {
+                                  minLength: 1,
+                                  maxLength: 50,
+                                  type: 'string',
+                                },
+                                street2: {
+                                  minLength: 1,
+                                  maxLength: 5,
```

Comment:
Yep, we were actually requested to make it `maxLength: 5` per our requirements. I believe it was because of the size of the field in the form as well.